src/graph.js

```diff
@@ -1,8 +1,64 @@
 /* eslint-disable */
-class Graph {
+class Node {
```

Comment:
Nice idea creating another class to encapsulate some functionality instead of relegating it all to the Graph class.